core/resources/plugin-test.md

```diff
@@ -0,0 +1,9 @@
+# Default Configuration - test
+
+``` yaml $(test)
```

Comment:
`test` is a reaaaaaly generic word. I'm pretty sure I have things that use `--test` which would trigger that. 

can we find a better switch for that?

Reply:
ok

Reply:
Circling back on this, what kind of tests is this generator generating?  Maybe I can help come up with a more specific name?